okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -77,6 +83,11 @@ public InternalModule(ModuleManager modules, TenantManager tenantManager,
     logger.info("InternalModule starting okapiversion={}", okapiVersion);
   }
 
+  /**
+   * Return module descriptor for okapi itself.
+   * @param okapiVersion Okapi version; null and "0.0.0" will be assumed
```

Comment:
```suggestion
   * @param okapiVersion Okapi version; null is converted to "0.0.0"
```

Reply:
null is not converted.

Reply:
This code
```
public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
    String v = okapiVersion;
    if (v == null) {  // happens at compile time,
      v = "0.0.0";   // unit tests can just check for this
```
converts null to "0.0.0".

Reply:
null is not converted. The version is converted.

Reply:
null is replaced by "0.0.0"?